### PR TITLE
 docs: update bounties page

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

Updates and restructures the `Bounties` page.

image

## Types of changes

- [x] Docs (documentation only changes)

## Checklist

- [x] I have read the [CONTRIBUTING](/guidebook/contribution-guidelines/contributing.html) documentation
- [x] I have read the [WRITING DOCUMENTATION](/guidebook/contribution-guidelines/writing-documentation.html) guidelines

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/faq/bounties.md [hunk 1]
```diff
@@ -10,69 +10,98 @@ Thanks for your interest in contributing to the ARK Ecosystem! Please take a mom
...
+
+### Reward Tiers
+
+ - `Tier 1` - $100
+ - `Tier 2` - $50
+ - `Tier 3` - $25
+ - `Tier 4` - $10
+ - `Tier 5` - $5
+ - `Tier 6` - $1
+ - `Tier 0` - Custom Reward
```
Comment (faustbrian) [inline docs/faq/bounties.md]: This should be above tier 1.

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $10 USD._

**Pull request merged**